draft-ietf-httpbis-priority.md

```diff
@@ -150,64 +140,28 @@ in a revision of HTTP/2 {{HTTP2}}. However, in order to maintain wire
 compatibility, HTTP/2 priority signals are still mandatory to handle (see
 {{Section 5.3.2 of HTTP2}}).
 
-Clients can build RFC 7540 trees with rich flexibility but experience has shown
-this is rarely exercised. Instead they tend to choose a single model optimized
-for a single use case and experiment within the model constraints, or do nothing
-at all. Furthermore, many clients build their prioritization tree in a unique
-way, which makes it difficult for servers to understand their intent and act or
-intervene accordingly.
```

Comment:
This paragraph reflects reality, I don't see it as bashing. I would like to keep something along these lines but can live with an edited down version. The difficulty for servers in interpreting or operating with relationship-based trees is one core part of the extensible priorities scheme. Some of the following edits in this PR also present text that overlooks tree troubles, I suspect we editors might want to keep that. @kazuho any thoughts?

Reply:
I wasn't pointing to this in particular as bashing, but more the overall effect of the section.

I thought that I addressed the most pertinent points in the text about request relationships.  More importantly, I think that h2bis covers this topic adequately and we don't need redundant text about the many ways in which RFC 7540 failed.

Reply:
I think we can remove this sentence, by moving the point made by the last sentence below (please see my review comment below).

Regarding other sentences, I'm not sure if they are a problem related to Extensible Priorities, and do not have an argument against removing them. Also, I'd point out that the last paragraph of this PR might imply the discussion here; it says that rich flexibility was deemed unnecessary for the web use-case.